botnim/vector_store/vector_store_es.py

```diff
@@ -261,28 +264,16 @@ def delete_existing_files(self, context_, vector_store, file_names):
             return 0
 
     def update_tools(self, context_, vector_store):
-        # vector_store is now just the index name string
```

Comment:
Not sure what was wrong with this code - the vector_store parameter being the index name already encapsulating the bot/context/environment information.

Reply:
@noamoss ?

Reply:
revert and tackle the issue with 'vector_store' only, add num results

Reply:
not reverted, instead `create_search_tool` is called both from here and from sync.py 🤯

Reply:
- correctly using the bot and context information from the current instance,
- eliminated the duplicate tool creation

Reply:
not fixed, still not using the vector_store and rebuilding the index.